src/js/stores/AppsStore.js

```diff
@@ -146,12 +146,29 @@ function processApp(app) {
   return app;
 }
 
-function processApps(apps) {
-  return apps.map(function (app) {
-    return processApp(app);
+function processEmptyGroup(group) {
+  return Object.assign(processApp({id: group.id}), {
+    isGroup: true,
+    status: null
   });
 }
 
+function processAppsAndGroups(group, apps = []) {
```

Comment:
To clarify: the data returned by that method was "bent" (the `isGroup` trick) in a way that would be easily consumed by the AppListComponent's present-day implementation.